src/open_inwoner/templates/pages/profile/appointments.html

```diff
@@ -14,14 +14,16 @@ <h1 class="utrecht-heading-1" id="title">{% trans "Mijn afspraken" %}</h2>
                 <div class="card card--compact card--stretch appointment-info">
```

Comment:
If you view this page in tablet-size it spreads the cards across one giant column - on 'Mijn aanvragen' we do not have this because the cardgrid is surrounded by the "cases__grid" class, so I guess we need to promote this class to a more generic "cards__grid" or something, that you can then use to control responsiveness by way of media-queries like the one below. 
But... maybe that is a 'helicopter level' styling issue that falls outside of the scope of this PR.

```(min-width: 500px) and (max-width: 767px)
.cards__grid .grid .column {
    flex-basis: calc(50% - var(--spacing-large));
}```